src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/TagHelperResolutionResultComparer.cs

```diff
@@ -36,9 +36,9 @@ public int GetHashCode(TagHelperResolutionResult obj)
 
             if (obj.Descriptors != null)
             {
-                for (var i = 0; i < obj.Descriptors.Count; i++)
```

Comment:
I assume this was done previously to avoid allocating an Enumerator, and that now that we have a HashSet an Enumerator is the only option?

Reply:
Yup!